src/NCMenuBar.cc

```diff
@@ -193,13 +187,29 @@ NCMenuBar::wHandleInput(wint_t key)
 
     switch (key) {
```

Comment:
Please move the opening brace to a separate line so it visually matches the closing brace.